react/features/recent-list/components/AbstractRecentList.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+
+import { Component } from 'react';
+import { ListView } from 'react-native';
+
+import { getRecentRooms } from '../functions';
+
+import { appNavigate } from '../../app';
+
+/**
+ * The type of the React {@code Component} state of {@link AbstractRecentList}.
+ */
+type State = {
+
+    /**
+     *  The {@code ListView.DataSource} to be used for the {@code ListView}.
+     *  Its content comes from the native implementation of
+     *  {@code window.localStorage}.
+     */
+    dataSource: Object
+}
+
+/**
+ * The type of the React {@code Component} props of {@link AbstractRecentList}
+ */
+type Props = {
+
+    /**
+     * Redux store dispatch function.
+     */
+    dispatch: Dispatch<*>,
+}
+
+/**
+ * Implements a React {@link Component} which represents the list of
+ * conferences recently joined, similar to how a list of last dialed
+ * numbers list would do on a mobile
+ *
+ * @extends Component
+ */
+export default class AbstractRecentList extends Component < Props, State > {
```

Comment:
Style: I believe we don't use spaces around < >

Reply:
Fixed, thanks!